nservicebus/upgrades/externalize-encryption.md

```diff
@@ -36,4 +35,23 @@ snippet: SplitDefiningEncryptedPropertiesAs
 
 ## Custom encryption Service
 
-snippet: SplitEncryptionFromIEncryptionService
\ No newline at end of file
+snippet: SplitEncryptionFromIEncryptionService
+
+
+## Compatibility
+
+The NServiceBus.Encryption.MessageProperty package is partially compatible with endpoints using NServiceBus core's encryption functionality:
+
+
+### Sending and receiving using NServiceBus.Encryption.MessageProperty
+* NServiceBus.Encryption.MessageProperty can de- and encrypt all messages with message properties of type `NServiceBus.WireEncryptedString`.
+* NServiceBus.Encryption.MessageProperty can de- and encrypt all messages with message properties of type `NServiceBus.Encryption.MessageProperty.WireEncryptedString`.
+* NServiceBus.Encryption.MessageProperty can de- and encrypt all messages using an encrypted property convention.
+
+
+### Sending and receiving using NServiceBus core
```

Comment:
Given the previous section is titled "Sending and receiving using NServiceBus.Encryption.MessageProperty", I think the context is already set for "NServiceBus" to mean the `NServiceBus` package, so I think "core" can probably be omitted here.